extensions/amp-date-picker/1.0/test/validator-amp-date-picker.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
Hi, thanks for your contribution. 

Seems like the PR is missing a few validation related files, specifically a .protoascii file and a /test/validator-*.out file.

The .protoascii file should describe the validation rules, the .html file is a test sample, and .out is the expected output of validation. 

Please add those files. In the .html file consider adding a few blocks that will fail validation in all possible ways worth testing

See https://github.com/ampproject/amphtml/blob/main/docs/component-validator-rules.md for details.

Feel free to browse other extensions for examples of validation rules and tests:  https://github.com/ampproject/amphtml/tree/main/extensions.

Reply:
Hi @MichaelRybak, I just checked in and confirmed with @jridgewell that these changes aren't necessary for this PR. The work in this PR supports Preact and Bento mode, but not AMP mode for right now.

Reply:
@becca-bailey Got it. What about the test/validator*.html file? Is it used in some way in this PR or should it be part of the AMP-related PR?

Reply:
At the moment it sounds like there are some open questions about the future of these auto-generated AMP files. The current advice I have received is to leave them alone for now, and they will most likely be removed in a future PR once we know for sure that we will not be supporting AMP mode for bento components.

Reply:
Any thoughts here @dethstrobe ?

Reply:
It does look like this file is causing some issues in CI. so we might just want to remove it.

Reply:
Removing it sounds good to me.

Reply:
This file was causing some issues in CI. so I just deleted it for now.